docs/pull_request_template.md

```diff
@@ -6,15 +6,10 @@
 
 If you're unsure about a box, leave it unchecked. A maintainer will help you.
 
-If a box isn't applicable, add an explanation in **bold**.
-For example: **(N/A: this is a bugfix, not a feature)**
-
 - [ ] I understand README.md. I also understand that acceptance of
   community PRs will be delayed until the test and CI systems are online.
-- [ ] If this is a feature addition, that feature has been voted into the
-  C++ Working Draft.

```

Comment:
Are you not concerned that this will open the door to a raft of suggestions for extensions?

Reply:
We've already got a checkbox for "I understand README.md" which has a prominent [Non-Goals](https://github.com/microsoft/STL#non-goals) section, explaining that we won't merge non-Standard extensions, or proposed features before they've been voted into the Working Paper.

Reply:
I suppose I'm naive to hope that people who will check "I understand README.md" without looking at `README.md` will pay any more attention to other bullets in this list.